ios/Inji.xcodeproj/project.pbxproj

```diff
@@ -368,10 +368,10 @@
 					"-lc++",
 				);
 				OTHER_SWIFT_FLAGS = "$(inherited) -D EXPO_CONFIGURATION_DEBUG";
-				PRODUCT_BUNDLE_IDENTIFIER = io.mosip.inji.collab;
+				PRODUCT_BUNDLE_IDENTIFIER = io.mosip.inji.mobileid;
```

Comment:
this change is required for the sed script to work in a sane manner as it's replacing all instances of `mobileid` with the flavor's correct suffix

Reply:
Maybe a comment above this would be good to state it's importance for the Fastlane & CI.